runner.yml.dist

```diff
@@ -47,7 +47,6 @@ commands:
         $settings["poetry.service.password"] = getenv('POETRY_SERVICE_PASSWORD');
         $settings["poetry.notification.username"] = getenv('POETRY_NOTIFICATION_USERNAME');
         $settings["poetry.notification.password"] = getenv('POETRY_NOTIFICATION_PASSWORD');
-        $config['tmgmt.translator.poetry']['settings']['service_wsdl'] = 'http://localhost:8080/build/poetry-mock/wsdl';
     # Generate settings.testing.php, it will be used when running functional tests.
```

Comment:
This is being taken out from here and set only for behat tests.
As a result, when doing manual tests, we have now an error when sending the request.

Reply:
Yes, we cannot keep that there anymore because it overrides the setting we are putting in Behat. So for manual testing, we need to set that in the settings file ourselves.